lib/settings_helper.rb

```diff
@@ -35,7 +35,7 @@ def self.relays_enabled_for_reservation?
   #   If you want to check setting 'feature.password_update_on'
   #   then this parameter would be :password_update
   def self.feature_on?(feature)
-    !!Settings.feature["#{feature}_on"]
+    !!Settings.feature[feature]
```

Comment:
Since we’re now just checking the hash key, what do you think about using `Settings.fetch(feature)` here, so we get an exception if we somehow try to check a setting that has not been configured? That way, it will be more obvious then getting back an unexpected result…

P.S. I think the comment above this method should be removed, since it’s no longer true 😄

Reply:
![e5e29239-a786-431d-b285-3e4640f6c20b](https://user-images.githubusercontent.com/1099111/50170707-2c2d9900-02b6-11e9-82bd-7b51676977f4.png)

Actually, using `fetch` is a great idea.